core/platform/auth/firebase_auth_services.py

```diff
@@ -116,11 +116,17 @@ def establish_auth_session(request, response):
     fresh_cookie = firebase_auth.create_session_cookie(
         _get_id_token(request), feconf.FIREBASE_SESSION_COOKIE_MAX_AGE)
 
+    import webapp2
+    response = webapp2.Response()
     response.set_cookie(
         feconf.FIREBASE_SESSION_COOKIE_NAME,
         value=fresh_cookie,
         max_age=feconf.FIREBASE_SESSION_COOKIE_MAX_AGE,
         overwrite=True,
+        # Only allow this cookie to persist on Oppia's requests and responses,
+        # promoting it to a "first-party cookie" (as opposed to a "third-party
+        # cookie", which are untrusted by some browsers).
+        samesite='lax',
```

Comment:
I think this is the default for modern browsers: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite

Reply:
Yup, have increased it to strict.